build.gradle

```diff
@@ -106,18 +107,6 @@ dependencies {
 
     annotationProcessor group: 'com.google.auto.value', name: 'auto-value', version: '1.7.3'
     annotationProcessor group: 'org.springframework.boot', name: 'spring-boot-configuration-processor'
-
-    // Force log4j version to at least 2.15.0 for security concerns.
-    implementation('org.apache.logging.log4j:log4j-api') {
```

Comment:
why are we not changing it to 2.16 here like what you did in the other repo?

Reply:
This does bump us to 2.16.0, but this adds the constraint via the Spring dependency manager plugin (https://docs.spring.io/spring-boot/docs/2.6.1/gradle-plugin/reference/htmlsingle/#managing-dependencies.dependency-management-plugin.customizing) instead of directly. 
Mostly it's just less verbose, the end effect is the same here